docs/integration.rst

```diff
@@ -17,7 +17,6 @@
 
 Integration
 ===========
-
```

Comment:
Please do not include this change.

Reply:
Thanks, How can i exclude it? there is option 'delete file' but will it delete file from the repo?

Reply:
Reset your commit and remove this file from being committed. https://stackoverflow.com/a/15321456

Reply:
And please check what you commit before you push :) `git status`

Reply:
Hi, I am sorry but reseting the commit and removing file doesnt make sense to me somehow (As reverting to some commiy will still have the things that you wanted me to remove) and eventually make things complicated. Can i checkout this file from apache/master and comit that?

```
git checkout apache/master -- docs/integration.rst
git commit -m "removing a file from PR"
git push origin master
```